apps/wallet-mobile/src/features/WalletManager/network-manager/network-manager.ts

```diff
@@ -72,6 +72,7 @@ export const protocolParamsPlaceholder = freeze({
   coinsPerUtxoByte: '4310',
   poolDeposit: '500000000',
   keyDeposit: '2000000',
+  epoch: 509,
```

Comment:
Dangerous, I assume we don't actually use this and is just there to satisfy the type

Reply:
it's actually safe, it can be the last epoch before voting begins (static), if the request to params fails between epochs and it actually needs to fallback definitely a some warning about using outdated values are a good idea though.